lib/Module/Starter/Simple.pm

```diff
@@ -768,16 +768,16 @@ WriteMakefile(
     ABSTRACT_FROM    => '$main_pm_file',
     LICENSE          => '$slname',
     PL_FILES         => {},
-    MIN_PERL_VERSION => $self->{minperl},
+    MIN_PERL_VERSION => '$self->{minperl}',
```

Comment:
This is within the heredoc for generating a Makefile.PL; $self->{minperl} is a variable from outside.